SR2023/2022-12-20-rules-update.md

```diff
@@ -0,0 +1,15 @@
+---
+to: Student Robotics 2023 teams
+subject: "Rules update: Bronze token masses"
+---
+
+Hi,
+
+We're pleased to announce that we've published an update to the rules which
+specifies the mass of the Bronze tokens. Their mass is specified as 80 ±40g.
+The mass of Gold tokens remains unspecified.
+
+Hopefully this will allow teams to progress on mechanisms to detect the
+difference between the various token types.
+
+-- the SR Team
```

Comment:
```suggestion
-- the Competition Team
```
Think this is how other emails have been signed off, not bothered either way though

Reply:
We've done a mix of both, depends a lot on who the author is I think.